fem/bilinearform.cpp

```diff
@@ -960,6 +958,18 @@ void BilinearForm::EliminateVDofsInRHS(
    mat->PartMult(vdofs, x, b);
 }
 
+void BilinearForm::Mult(const Vector &x, Vector &y) const
```

Comment:
Don't we need similar changes to `BilinearForm::AddMult()` to support `ext`?

Reply:
Do you want me to add them? We did this in the Mixed case because they were needed for some operations.

Reply:
I would suggests adding them because it doesn't seem like much effort.  However, I won't object if you postpone this and start an issue so we won't forget to address it.

Reply:
I'd prefer to postpone this to a different PR.